torch/_subclasses/fake_tensor.py

```diff
@@ -1170,6 +1170,10 @@ def is_our_fake(self, t: object) -> TypeGuard[FakeTensor]:
     #   (see NOTE: [torch.tensor, lift_fresh, and device movement])
     @property
     def avoid_device_init(self) -> bool:
+        # if torch.xpu._is_compiled():
```

Comment:
doesnt look ready for review

Reply:
Sorry, I was sloppy and forgot to remove the comments.